src/helper/convertColumnElementsToData.js

```diff
@@ -11,92 +11,69 @@
 
```

Comment:
So I refactored this file.

`convertColumnElementsToData` is a helper, and thus now only extracts props and templates.

The logic where this helper is used to construct the props/templates through React children or Column Data is moved over to a new selector called `columnData`